uconnrcmpy/conditions.py

```diff
@@ -851,6 +851,15 @@ def compare_to_sim(self, run_reactive=False, run_nonreactive=True):
             print_str += '{:.6f}'.format(ignition_delay)
             copy_str += '{}'.format(ignition_delay)
 
+        if self.nonreactive_sim is not None and self.reactive_sim is not None:
```

Comment:
Just want to point out (no changes necessary). You asked me where the `self.reactive_sim.temperature` comes from and if it will error if the reactive_sim has not run. The answer is no, because of this line. This is checking if the `nonreactive_sim` **and** the `reactive_sim` have been added.